<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dashboard::CDashboardWidget2</class>
 <widget class="QWidget" name="Dashboard::CDashboardWidget2">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_2" stretch="0,1">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="margin">
    <number>0</number>
   </property>
   <item>
    <widget class="CProgramPanelWidget" name="programPanelWidget" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
    </widget>
   </item>
   <item>
    <widget class="QStackedWidget" name="stackedWidget">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="CContainerPanelWidget" name="containerPanelWidget"/>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>CProgramPanelWidget</class>
   <extends>QWidget</extends>
   <header>Dashboard/Include/ProgramPanelWidget.h</header>
   <container>1</container>
  </customwidget>
  <customwidget>
   <class>CContainerPanelWidget</class>
   <extends>QWidget</extends>
   <header>Dashboard/Include/ContainerPanelWidget.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections/>
</ui>
